src/main/java/frc/robot/subsystems/shooter/commands/bits/CheckShooterAccuracy.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems.shooter.commands.bits;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.Constants;
+import frc.robot.subsystems.shooter.Shooter;
+import frc.robot.subsystems.shooter.commands.Shoot;
+
+import java.util.OptionalDouble;
+import java.util.function.DoubleSupplier;
+
+public class CheckShooterAccuracy extends Shoot {
+    private final Shooter shooter;
+    private final DoubleSupplier distance;
```

Comment:
you can make the shooter and the distance `protected` and then you won't need to store them twice.

Reply:
You still can do it

Reply:
I don't understand

Reply:
Make the shooter, distance and power `protected` in `Shoot` command, and then you can don't need to store the fields again